tests/plugins/widget/integration/wysiwygarea/manual/preventofremovingdiv.html

```diff
@@ -0,0 +1,22 @@
+<textarea id="editor1" name="editor1">
```

Comment:
This test fails on IE8 due to `addEventListener`, if it should be run only for Edge add a proper `bender.ignore` or fix attaching listener if it should be run also for IE8.

Reply:
I added both things. I know it wasn't necessary, but this not harm anything. ;)